lib/puppet_litmus/rake_helper.rb

```diff
@@ -164,15 +164,20 @@ def configure_path(inventory_hash)
     results
   end
 
-  def build_module(opts)
-    # old cli_way
-    # pdk_build_command = 'bundle exec pdk build  --force'
-    # stdout, stderr, _status = Open3.capture3(pdk_build_command)
-    # raise "Failed to run 'pdk_build_command',#{stdout} and #{stderr}" if (stderr =~ %r{completed successfully}).nil?
-    require 'pdk/module/build'
-    require 'pdk/util'
+  # @param opts Hash of options to build the module
+  # @option opts [String] :module_dir The path of the module to build. If missing defaults to Dir.pwd
+  # @option opts [String] :target_dir The path the module will be built into. The default is <source_dir>/pkg
+  # @return [String] The path to the built module
+  def build_module(opts = {})
```

Comment:
Any particular reason you're using an option hash instead of explicit parameters? I think explicit parameters would make the code easier to read.

Reply:
was probably taken out from the pdk: https://github.com/puppetlabs/pdk/blob/master/lib/pdk/module/build.rb#L13